cmd/kubeadm/app/util/system/kernel_validator.go

```diff
@@ -75,7 +74,8 @@ func (k *KernelValidator) Validate(spec SysSpec) (error, error) {
 	errs = append(errs, k.validateKernelVersion(spec.KernelSpec))
 	// only validate kernel config when necessary (currently no kernel config for windows)
 	if len(spec.KernelSpec.Required) > 0 || len(spec.KernelSpec.Forbidden) > 0 || len(spec.KernelSpec.Optional) > 0 {
-		errs = append(errs, k.validateKernelConfig(spec.KernelSpec))
+		err := k.validateKernelConfig(spec.KernelSpec)
```

Comment:
lol. not sure why this got changed. will change it back in the morning. should be no change here.

Reply:
Can we revert the changes here? This does not affect the patch, but it's not relevant for the change.